### PR TITLE
Setup phpstan

### LINKED ISSUE #2: [Github Action] Setup phpstan

### PR DESCRIPTION
**Pull request by aryaantony92**
Resolves #2

### CODE REVIEW

File: phpstan.neon [hunk 1]
```diff
@@ -0,0 +1,10 @@
+parameters:
+    level: 5
+    reportUnmatchedIgnoredErrors: false
+    paths:
+        - src
+    bootstrapFiles:
+        - phpstan-bootstrap.php
+    excludePaths:
+        - src/Migrations/*
```
Comment (robertSt7) [inline phpstan.neon]:
```suggestion
```
@aryaantony92  I think the Migrations should also be checked. What do you think?

Comment (aryaantony92) [reply phpstan.neon]: Yes, we can do

File: .github/workflows/static-analysis.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+name: "Static Analysis"
+
+on:
+    pull_request:
+        branches:
+            - "[0-9]+.[0-9]+"
+            - "[0-9]+.x"
```
Comment (robertSt7) [inline .github/workflows/static-analysis.yaml:7]:
```suggestion
            - "[0-9]+.x"
        paths-ignore:
            - 'doc/**'
            - 'public/**'
```

File: .github/workflows/static-analysis.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    push:
+        branches:
+            - "[0-9]+.[0-9]+"
+            - "[0-9]+.x"
```
Comment (robertSt7) [inline .github/workflows/static-analysis.yaml:14]:
```suggestion
            - "[0-9]+.x"
        paths-ignore:
            - 'doc/**'
            - 'public/**'
```

**Review by robertSt7 — APPROVED** [review]

**Pull request merged**